### PR TITLE
Disable prepared statements.

### PR DESCRIPTION
**Pull request by simi**
- this is needed (at least temporary) for pgbouncer needed for database major version upgrade with (almost) zero downtime
- Are there any performance concerns regarding this? We will need to decide if that's ok to merge for now or if we should merge this temporarily before and revert after.
- for now I'm just checking if CI is happy

### CODE REVIEW

**Comment by martinemde** [comment]
should we test out having this deployed for a lower traffic hour (?) and then revert if it makes any difference?

**Comment by simi** [comment]
> should we test out having this deployed for a lower traffic hour (?) and then revert if it makes any difference?

not sure there are any lower traffic hours :(, we will need to find plan

**Comment by segiddins** [comment]
Is the change in encrypted database URLs intentional? I don't see why we'd need those to change to disable prepared statements

File: config/database.yml.sample [hunk 1]
```diff
@@ -2,6 +2,7 @@ default: &default
   adapter: postgresql
   encoding: utf8
   username: postgres
+  prepared_statements: false
```
Comment (segiddins) [inline config/database.yml.sample:5]: maybe source this from ENV so we can manually update the deployment in k8s if needed?

**Comment by simi** [comment]
> Is the change in encrypted database URLs intentional? I don't see why we'd need those to change to disable prepared statements

If I understand it well, when using `DATABASE_URL` database.yml is skipped at all. I have appended `?prepared_statements=false` to all addresses in encrypted database URLs.

**Review by segiddins — APPROVED** [review]

**Comment by simi** [comment]
I'm ok to merge this and deploy. @segiddins WDYT?

**Pull request merged**